config/initializers/devise.rb

```diff
@@ -18,13 +32,13 @@
 
   # ==> Controller configuration
   # Configure the parent class to the devise controllers.
-  # config.parent_controller = 'DeviseController'
+  config.parent_controller = 'TurboController'
 
   # ==> Mailer Configuration
   # Configure the e-mail address which will be shown in Devise::Mailer,
   # note that it will be overwritten if you use your own mailer class
   # with default "from" parameter.
-  config.mailer_sender = 'please-change-me-at-config-initializers-devise@example.com'
+  config.mailer_sender = 'manager@montecinema.com'
```

Comment:
I would store this email in env 🤔

Reply:
Sure, I'll leave that as it is for now and gonna get back to it when I prepare my app for deployment.